src/utils/monitoring.ts

```diff
@@ -206,10 +206,10 @@ export const getAccountFromNimbusKey = async (
     !authorMappingCache[nmbsKey] ||
     authorMappingCache[nmbsKey].lastUpdate < Date.now() - 3600 * 1000
   ) {
-    const mappingData = (await api.query.authorMapping.mappingWithDeposit(nmbsKey)) as Option<any>;
+    const mappingData = (await api.query.authorMapping.mapping(nmbsKey)) as Option<any>;
```

Comment:
Where does this come from ? Did we change the storage item in our AuthorMapping pallet ?

Reply:
Good question. I remember being confident about the change when I made it, but now I can't seem to figure that out again. It looks like it should still be `mappingWithDeposit`...

Reply:
This seems wrong to me, so I made an issue to track it. Thanks for questioning it.

Reply:
No wories. I pushed a fix in main.